Hymns/Display/DisplayHymnToolbarView.swift

```diff
@@ -15,28 +15,22 @@ struct DisplayHymnToolbarView: View {
                 Text("LYRICS").foregroundColor(self.toolbarTab == ToolbarTabs.lyrics ? .accentColor : .primary).onTapGesture {
                     self.toolbarTab = ToolbarTabs.lyrics
                 }
-                viewModel.chordsUrl.map { _ in
                     HStack {
                         Spacer()
                         Text("CHORDS").foregroundColor(self.toolbarTab == ToolbarTabs.chords ? .accentColor : .primary).onTapGesture {
                             self.toolbarTab = ToolbarTabs.chords
                         }
                     }
-                }
-                viewModel.chordsUrl.map { _ in
                     HStack {
                         Spacer()
                         Text("GUITAR").foregroundColor(self.toolbarTab == ToolbarTabs.guitar ? .accentColor : .primary).onTapGesture {
                             self.toolbarTab = ToolbarTabs.guitar
                         }
                     }
-                }
-                viewModel.chordsUrl.map { _ in
                     HStack {
                         Spacer()
                         Text("PIANO").foregroundColor(self.toolbarTab == ToolbarTabs.piano ? .accentColor : .primary).onTapGesture {
                             self.toolbarTab = ToolbarTabs.piano
```

Comment:
What loading issue does this cause? They're necessary because you don't want to show a "guitar" option if there is no guitar music to show. I map over the optional so that if the optional is nil, then it won't show that item

Reply:
Some of the HStack was coming in slow. So lyrics would immediately be there and then later the others would appear as they were mapped in. Do you do that on the Android app as well? Having the tabs just not show up if there is no music sheets?

Reply:
yup, on Android I don't show the buttons if there is no music for that song. I wonder why they're coming slow. It didn't seem slow when I tried it

Reply:
Okay, it is looking good now...